app.json

```diff
@@ -2,7 +2,7 @@
   "expo": {
     "name": "TreeMapper",
     "slug": "treemapper",
-    "version": "2.0.1",
+    "version": "2.0.3",
```

Comment:
Don't forget to update this number also in the iOS project and Android build file for the native apps.

Reply:
If expo is generating these file on the fly and using this info for it, it probably isn't necessary any more to care about this manually.

Reply:
Yes